### PR TITLE
DOCS-952: Update baud rate

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/movement-sensor/gps/gps-nmea.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Edit and fill in the attributes as applicable.
       "attributes": {
         "connection_type": "serial",
         "serial_attributes": {
-          "serial_baud_rate": 115200,
+          "serial_baud_rate": 38400,
```
Comment (npentrel) [inline docs/components/movement-sensor/gps/gps-nmea.md:76]:
```suggestion
          "serial_baud_rate": 115200,

```

File: docs/components/movement-sensor/gps/gps-nmea.md [hunk 2]
```diff
@@ -74,6 +74,7 @@ Edit and fill in the attributes as applicable.
         "connection_type": "serial",
         "serial_attributes": {
           "serial_baud_rate": 115200,
+
```
Comment (npentrel) [inline docs/components/movement-sensor/gps/gps-nmea.md:77]:
```suggestion
```

**Review by susmitaSanyal — CHANGES_REQUESTED** [review]
`serial_baud_rate` default is also 38400

**Review by susmitaSanyal — APPROVED** [review]
LGTM

**Pull request merged**